src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -38,7 +38,7 @@ namespace Microsoft.IdentityModel.Protocols
     /// <summary>
     /// Manages the retrieval of Configuration data.
     /// </summary>
-    /// <typeparam name="T">must be a class.</typeparam>
+    /// <typeparam name="T">The type of element. It must be a class.</typeparam>
```

Comment:
change: 'The type of element.' to 'The type of the <see cref="IDocumentRetriever"/>